ext/Pod-Html/t/lib/Testing.pm

```diff
@@ -2,11 +2,13 @@ package Testing;
 use 5.10.0;
 use warnings;
 require Exporter;
-our $VERSION = 1.26; # Let's keep this same as lib/Pod/Html.pm
+our $VERSION = 1.27_001; # Let's keep this same as lib/Pod/Html.pm
```

Comment:
I'm not sure what this version is for exactly but this won't work as intended. Underscores in numeric literals don't become part of the value. Versions with underscores must be strings (and versions should always be strings anyway, but that's not your fault).

Reply:
This was corrected in https://github.com/Perl/perl5/pull/18950/commits/0e756f8ee9402244f1edce6a0a4cb47e323f4121 shortly after the p.r. was filed.